test/integration/document/Thumbnails.e2e.test.js

```diff
@@ -280,4 +280,17 @@ describe('Preview Document Thumbnails', () => {
         cy.contains('Single Page Document');
         cy.getByTestId('thumbnails-sidebar').should('not.be.visible');
     });
+
+    it('Should change page on up and down arrow keyboard shortcuts', () => {
+        showDocumentPreview({ enableThumbnailsSidebar: true });
+
+        getThumbnailWithRenderedImage(2).click();
+        cy.focused().type('{downarrow}');
+        getThumbnailWithRenderedImage(3).should('have.class', THUMBNAIL_SELECTED_CLASS);
+
+        getThumbnailWithRenderedImage(3).click();
+        cy.focused().type('{uparrow}');
+
+        getThumbnailWithRenderedImage(2).should('have.class', THUMBNAIL_SELECTED_CLASS);
```

Comment:
should we also verify the behavior that the page being previewed changes with the keyboard button presses?

Reply:
added